spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -1943,12 +2047,15 @@ The `spring-cloud-stream-schema` module contains two types of message converters
 === Converters with Schema Support
 
 The `AvroSchemaMessageConverter` supports serializing and deserializing messages either by using a predefined schema or by using the schema information available in the class (either reflectively or contained in the `SpecificRecord`).
-If the target type of the conversion is a `GenericRecord`, a schema must be set.
+If the user provides a custom converter (as below), then the default `AvroSchemaMessageConverter` bean will not be created.
```

Comment:
Much clearer now!

Reply:
It is much better. To be consistent with the rest of the document, though, I'd change it to: 

If you provide a custom converter, then the default `AvroSchemaMessageConverter` bean is no created. The following example shows a custom converter:

Thanks for putting up with my nitpicking. :)